content/blog/2016/2016-04-07-jenkins-community-survey-results-blog.adoc

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Jenkins Community Survey Results: CD, Pipelines, and Star Wars"
+tags:
+- jenkins
+- survey
+- continuous delivery
+- pipelines
+- docker
+- star wars
+author: bvdawson
+---
+
+*Jenkins Community Survey Results –Continuous Delivery, Pipelines and
+Star Wars*
+
+image:/images/blog/2016-survey-blog-butler.png[image,width=84,height=115]Last
+fall, at the Jenkins User Conference – US West (JUC), the community
+asked JUC attendees to take a survey. Almost 250 of you did – and thanks
```

Comment:
JUC attendees? Is this not [the survey that KK posted on the blog](https://jenkins.io/blog/2015/09/01/take-the-2015-jenkins-survey/)?